src/components/UI/DashBoard_Catalog.jsx

```diff
@@ -1,11 +1,13 @@
-import { Link } from "react-router-dom";
 import { NavBar } from "./NavBar";
+import { Link } from "react-router-dom";
 
 const Catalog = (props) => {
+  const { party } = props;
   return (
```

Comment:
Good practice to de-structure all the props everywhere. 
It helps with the readability of our codebase. 👍